cmd/monaco/generate/deletefile/command.go

```diff
@@ -47,14 +48,15 @@ func Command(fs afero.Fs) (cmd *cobra.Command) {
 				return err
 			}
 
-			return createDeleteFile(fs, manifestName, projects, environments, fileName, outputFolder)
+			return createDeleteFile(fs, manifestName, projects, environments, fileName, excludeTypes, outputFolder)
```

Comment:
Opinion: In terms of UX I find an "exclude" flag a bit weird personally. I get that there might a be need (like this one) to say "delete all but", but I would expect that the inverse "include/only" flag exists as well so I can say "remove all of this type configured via this project".

What was the existing request for this feature?

Reply:
i agree, the opposite would be more useful. Why not offer both? :)
Let me explain my reasoning. First i wanted to implement the filtering in the integration test cleanup only. However, it turns out that code cannot easily be reused for that purpose. Of course the "read-unmarshalToMap-modify-marshalFromMap-write" logic can be written manually, and I did that. However, there is the risk that this logic would silently fail if delete file structure slightly changes, and all of the sudden we have a failing test and someone needs to dig in why...

so i thought, why not introduce that flag, it might be not THE usecase everyone is searching for but hey it does no harm right?

Anyway, do you want me to revert that commit? :)

Reply:
as mentioned via DM - I'd suggest you discuss whether to keep, extend or revert the feature in the whole team next week. 

mentioned my alternative idea on how to ensure the setup for e2e tests in dm